src/Calibur.sol

```diff
@@ -87,7 +87,7 @@ contract Calibur is
     function executeUserOp(PackedUserOperation calldata userOp, bytes32) external onlyEntryPoint {
         // Parse the keyHash from the signature. This is the keyHash that has been pre-validated as the correct signer over the UserOp data
         // and must be used to check further on-chain permissions over the call execution.
-        (bytes32 keyHash,,) = abi.decode(userOp.signature, (bytes32, bytes, bytes));
+        (bytes32 keyHash,,) = userOp.signature.decodeWithKeyHashAndHookData();
```

Comment:
This needed to be changed in #187 where we decode signatures in calldata now. It was possible to send in a wrapped signature with:
`keyHash || signature` but no hookData which would not revert in `validateUserOp` but would revert here when you did the in memory decoding

Reply:
This doesn't NEED to be decoded in calldata afaik.. the reason we needed to decode in 187 is to prevent short circuiting but this is like a top level function call from the entrypoint

if it reverts here it just means it was malformed calldata

Reply:
We are changing this behaviour slightly with https://github.com/Uniswap/calibur/pull/218